libr/debug/debug.c

```diff
@@ -1219,6 +1219,70 @@ R_API int r_debug_continue_until_nonblock(RDebug *dbg, ut64 addr) {
 	return r_debug_continue_until_internal (dbg, addr, false);
 }
 
+R_API bool r_debug_continue_back(RDebug *dbg) {
+	RDebugSession *before;
+	RBreakpointItem *prev = NULL;
+	int has_bp;
+	ut64 pc, end_addr;
+
+	if (r_debug_is_dead (dbg))
+		return false;
+	if (!dbg->anal || !dbg->reg) {
```

Comment:
Probably this should be before `r_debug_is_dead()` and also check `dbg` pointer too.